src/components/OSCALCatalogGroups.js

```diff
@@ -0,0 +1,102 @@
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import List from "@mui/material/List";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import { styled } from "@mui/material/styles";
+import Tab from "@mui/material/Tab";
+import Tabs from "@mui/material/Tabs";
+import PropTypes from "prop-types";
+import React from "react";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import OSCALCatalogGroup, { getGroupKey } from "./OSCALCatalogGroup";
+import OSCALControlParamLegend from "./OSCALControlParamLegend";
+
+export const OSCALControlList = styled(List)`
+  padding-left: 2em;
+  padding-right: 2em;
+`;
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`vertical-tabpanel-${index}`}
+      aria-labelledby={`vertical-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
+
+const ComponentTabs = styled(Tabs)(
+  ({ theme }) => `
+  border-right: 1px solid ${theme.palette.divider};
+`
+);
+
+function a11yProps(index, title) {
+  return {
+    id: `vertical-tab-${index}`,
+    "aria-controls": `vertical-tabpanel-${title}`,
+  };
+}
+
+TabPanel.propTypes = {
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+export default function OSCALCatalogGroups(props) {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <OSCALSection>
+      <Card>
+        <CardContent>
+          <Grid container>
+            <Grid item sm={9}>
+              <OSCALSectionHeader>Control Groups</OSCALSectionHeader>
+            </Grid>
+            <Grid item sm={3}>
+              <Box display="flex" justifyContent="flex-end">
+                <OSCALControlParamLegend />
+              </Box>
+            </Grid>
+            <Grid item sm={2}>
+              <ComponentTabs
+                onChange={handleChange}
+                orientation="vertical"
+                variant="scrollable"
+                value={value}
+              >
+                {props.groups?.map((group, index) => (
+                  <Tab
+                    key={group.title}
+                    label={group.title}
+                    {...a11yProps(index, group.title)}
+                  />
```

Comment:
Can you try to follow https://mui.com/material-ui/react-tabs/#customization to make it more like the appearance in the ticket. Specifically, I think I'd like to try to:

 1. Set `text-transform: none` so that we don't uppercase everything
 2. Try to use slightly better color choices
 3. Try and to `text-alight: left` or something else reasonable (this may require padding/margin)